editor/src/clj/editor/font.clj

```diff
@@ -586,8 +586,8 @@
                      :render-mode (:render-mode font-map)
                      :all-chars (:all-chars font-map)
                      :characters (:characters font-map)
-                     :cache-width (:cache-width font-map)
-                     :cache-height (:cache-height font-map)
+                     :cache-width cache-width
+                     :cache-height cache-height
```

Comment:
The default values are 0, which means, "grow dynamically" at runtime.
If the user wishes a certain size, they can set the values explicitly.